### DISCUSSION TITLE
Allow consent checkboxes to be checked by default

### DISCUSSION THREAD

**Discussion by ymarkus** [studio1902/statamic-peak-seo#30] [Ideas]
I think it would be nice to have this feature. Otherwise basically no user selects all checkboxes one by one and a lot of function is lost!
I have it working on my website and have a PR ready. 
It will still be GDPR-compliant, as the user can click "ignore" to not accept any. 
I already debugged this with google tag assistant.

Is this a feature you are interested in?

**Comment by robdekort** [comment]
Hey thanks for your message. I do think this isn’t valid as pre-checking is considered a dark pattern. Like you say: otherwise users don’t consent. So the goal here is to trick folks into giving consent. Yet when they’re presented the option as is, users clearly do not want to be tracked when given the option. 

However I recently had to implement this as well for a client and I don’t really want to fight marketers anymore. I’ve added it to my list and will PR in this change, together with some more changes, soon. Good for you?

thanks!

**Reply by ymarkus** [reply]
Thanks for the quick reply. I'm not a big fan of having tracking myself.
It's not so much tricking people, but 95% of users don't even read these cookie banners and blindly click. Even though they wouldn't mind tracking (as it is with a large part of the population) these aren't tracked.
Do you want me to send my PR or will you do it yourself?

**Reply by robdekort** [reply]
I’ll do it myself. Thanks. Or possibly not pre-checking but an accept all button. Would that work for you as well?

**Reply by robdekort** [reply]
Example here: https://facitdatasystems.com/

**Reply by ymarkus** [reply]
Yes, perfect! Thank you.

**Reply by robdekort** [reply]
Cool. Will implement this into the seo addon then. As an option.
